bukkit/src/main/java/dev/aurelium/auraskills/bukkit/requirement/RequirementListener.java

```diff
@@ -163,4 +167,22 @@ private void checkItemRequirements(Player player, ItemStack item, Cancellable ev
         }
     }
 
+    private void checkBlockRequirements(Player player, Material material, Cancellable event) {
+        BlockRequirement blockRequirement = manager.getBlocks().stream().filter(b -> b.getMaterial() == material).findFirst().orElse(null);
```

Comment:
Using a normal for loop instead of streams is preferred for slightly better performance, especially since this method could be called frequently.